qiita_db/metadata_template/constants.py

```diff
@@ -70,6 +70,15 @@
 ALL_RESTRICTIONS = [SAMPLE_TEMPLATE_COLUMNS, PREP_TEMPLATE_COLUMNS,
                     PREP_TEMPLATE_COLUMNS_TARGET_GENE]
 
+# This is what we consider as "NaN" cell values on metadata import
+NA_VALS = ['', 'no_data', 'unknown', 'Unspecified', 'unspecified']
```

Comment:
I think this this is a change needed for consistency in the variable names. BTW you mentioned that """Ok, left. Not changing the NA_VALS variable name though, since the actual pd.read_csv function calls it na_values""" but didn't keep it for the others ...